java/src/lib.rs

```diff
@@ -62,7 +62,7 @@ fn redis_value_to_java<'local>(env: &mut JNIEnv<'local>, val: Value) -> JObject<
             hashmap
         }
         Value::Double(float) => env
-            .new_object("java/lang/Double", "(D)V", &[float.into_inner().into()])
```

Comment:
Why it didn't cause issues before, but started now?

Reply:
because of https://github.com/amazon-contributing/redis-rs/pull/127